web/tests/upload.test.js

```diff
@@ -50,8 +50,8 @@ test("get_item", () => {
         $("#compose_banners .upload_banner.file_id_1 .upload_msg"),
     );
     assert.equal(
-        upload.get_item("upload_banner_close_button", {mode: "compose"}, "id_2"),
-        $("#compose_banners .upload_banner.file_id_2 .compose_banner_close_button"),
+        upload.get_item("upload_banner_cancel_button", {mode: "compose"}, "id_2"),
+        $("#compose_banners .upload_banner.file_id_2 .upload_banner_cancel_button"),
```

Comment:
What are we doing about tests for `compose_banner_close_button` here, I can see only tests for `upload_banner_cancel_button`?

Reply:
Added the tests for `upload_banner_close_button` also @yogesh-sirsat could you please review the PR, thanks :)

Reply:
I still don't see the tests for `upload_banner_close_button`.

I have reviewed the PR and looks fine except for this above tests